fluentui_listitem/src/main/java/com/microsoft/fluentui/tokenized/tabItem/TabItem.kt

```diff
@@ -191,13 +205,13 @@ fun TabItem(
                 }
             }
         }
-
+        val cornerRadiusSize = FluentGlobalTokens.CornerRadiusTokens.CornerRadiusCircle.value
+        val indicatorWidth = FluentGlobalTokens.SizeTokens.Size160.value
         Column(
             horizontalAlignment = Alignment.CenterHorizontally,
             verticalArrangement = Arrangement.Center,
             modifier = modifier
                 .then(clickableModifier)
-                .background(backgroundColor)
```

Comment:
Why is this removed?

Reply:
to remove the background black shadow that appears on clicking the icon